docker/sink-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
   sink-server:
     # Using an older version as the latest image broke our build.
     # image: "hapiproject/hapi:latest"
-    image: "hapiproject/hapi:v6.1.0"
+    image: "hapiproject/hapi:latest"
```

Comment:
Given that this works now, please remove the above two comment lines.

Reply:
done